addons/website_event/static/src/js/website_event.js

```diff
@@ -100,6 +103,40 @@ var EventRegistrationForm = publicWidget.Widget.extend({
             })
         });
     },
+
+    _addTurnstile: function (form) {
+        if (!session.turnstile_site_key) {
+            return false;
+        }
+        this._removeTurnstile();
+        const div = document.createElement("div");
+        div.classList.add("s_turnstile");
+        div.classList.add("cf-turnstile");
+        div.classList.add("float-end")
+        div.setAttribute("data-sitekey", session.turnstile_site_key);
+        div.setAttribute("data-appearance", "interaction-only");
+        div.setAttribute("data-appearance", "always");
+        div.setAttribute("data-response-field-name", "turnstile_captcha");
+        div.setAttribute("data-action", "recaptcha_token_response");
+        div.setAttribute("data-error-callback", "(code) => throw new Error('Turnstile error: '+ code)");
+
+        const modalFooter = form.querySelector("div.modal-footer");
+        modalFooter.insertBefore(div, modalFooter.firstChild);
```

Comment:
This should be the same
```suggestion
        modalFooter.prepend(div);
```